deploy/helm/stac-scaler/templates/proc-stac-col-deploy.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: processor-stac-collection
+  namespace: pgstac
```

Comment:
This should be pulled from a variable (see previous comments about namespace).

Reply:
Changed to remove hardcoded value for namespace.